locales/en-US/faq.properties

```diff
@@ -2,38 +2,55 @@
 faq_header=Donor FAQ
 faq_donate_link=donate now
 faq_intro=Below are the most frequently asked questions about donating to the Mozilla Foundation. If your question and answer are not listed here, please contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+faq_intro_thunderbird=Below are the most frequently asked questions about donating to Thunderbird. If your question and answer are not listed here, please contact us at <a href="mailto:thunderbird-donate@mozilla.org">thunderbird-donate@mozilla.org</a>.
 faq_item_1_header=What are the ways I can donate?
 faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. For additional ways to give -- like check, bitcoin, or bank transfer -- visit our <a href="https://donate.mozilla.org/ways-to-give">Ways to Give page</a>.
+faq_item_1_paragraph_thunderbird=<a href="https://donate.mozilla.org/thunderbird/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. Unfortunately donations by wire transfer is <em>not</em> possible at the moment. We're hoping to make that option available in the near future.
 faq_item_2_header=How does Mozilla ensure my personal information is protected?
 faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/privacy/">Mozilla Privacy Policy</a>.
 faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">PayPal</a> or <a href="https://stripe.com/privacy/">Stripe</a>, those organizations also collect user data.
+faq_item_2_header_thunderbird=I wan't to donate by wire transfer. What is your IBAN number?
+faq_item_2_p_thunderbird_a=Unfortunately direct wire transfers from outside of the US are very expensive for us. Among other fees there is an additional fee of $15 for each incoming wire transfer, which can exceed the amount of the donation. Therefore we do not offer wire transfers as a payment method for the moment.
```

Comment:
@WillatMozFdn should we take this opportunity to add the same item to the Mozilla FAQ, or we need a different wording? That would solve the use case Stephan was mentioning, I believe